### DISCUSSION TITLE
中元节故事会 | /story/ghost-story/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#588] [Announcements]
并不怎么恐怖，胆小鬼也可以看。

https://yuanfan.rbind.io/story/ghost-story/

**Comment by yihui** [comment]
这不是讲故事，这是说相声，到处是包袱和梗。

穿青蛙装的四个学生是自来也、弥彦、长门、小南吗？

**Reply by earfanfan** [reply]
哈哈，我谢谢你的夸奖。话说，按照相声情节，“我”已经下去一趟把“你”捞上来了，现在是不是又可以快乐下蛋，把年更遥提上日程了呢？

>穿青蛙装的四个学生是自来也、弥彦、长门、小南吗？

为撒这么问？

**Reply by yihui** [reply]
自己下蛋太费劲，加上最近在努力搬砖，所以暂且还是先看别人下蛋吧，等砖搬得对得起良心了再回到我的鸡窝里下蛋。哦，还有，我的笔记本电池不行了，前天刚拿去修，可能要两周后才能拿回来，我的年更遥日志们都在那台电脑上，所以这两周估计也不会写什么了。

因为他们正好是[穿着青蛙装的四个人](https://www.bilibili.com/video/BV1Y4411A7st/)：

![image](https://github.com/earfanfan/yuanfan/assets/163582/8f1e470a-aed1-40ec-a475-a4439c41a31f)

**Reply by earfanfan** [reply]
哦，搬吧。就是久坐要注意坐直，不然会腰疼……

**Reply by yihui** [reply]
我的电脑椅靠着还算舒适，而且每天雷打不动二百四十个仰卧起坐，外加十分钟起步的平板支撑，以及上下午分别灌下一升水导致的数次厕所奔波（没看见高跟鞋），应该足以对付久坐带来的身体毛病了。我现在唯一的小困扰是手腕，在国内连续多天凶狠打球，导致反拍外旋发力时手腕有轻微痛感（回国之前就已经有端倪了）；往后在这边连续打四五小时球时我得克制一下发力了，让手腕休养休养，争取从力量型选手过渡到落点型选手。

**Reply by earfanfan** [reply]
>上下午分别灌下一升水

以升为单位灌水喝真得好好笑，让我想起了一部电影的[喝牛奶名场面](https://www.bilibili.com/video/BV1FJ411M7RC/)。超链狂魔得了个新绰号，灌水仙人，再加上以前攒的禾几大神，绝情坑主，哈哈哈。

你慢慢过渡吧，我最近连球都没得打了，下次打球得等到下个月了。不过如果休养都没能使痛感自愈的话，可能不只是一般的炎症了。运动确实也会损伤身体，我经常骑车就会膝盖疼，最近才意识到可能是骑车姿势哪里不对，每次都注意调整一下就好多了。

另，本篇相声里面其实大多取材于真实生活。比如，我真得有一次在公司上厕所看到隔壁有一只空的高跟鞋，给小花讲了一句就把她吓到了，下班路上看见一只青蛙在骑车，去烧纸钱真得烧了半小时，还真得被烟熏到了，在长江汉江交界处有个龙王庙，龙王庙附近有个轮渡码头叫做四官殿。啃鸭头吸脑髓是列弛提到的。金木水火土真得是火腿腿最长么

**Reply by earfanfan** [reply]
还有，昨晚半夜醒来，看时间正好是4:44……

**Reply by yihui** [reply]
当然只能是火了，因为只有火腿肠，没有金木水土腿肠。

**Reply by earfanfan** [reply]
>自己下蛋太费劲

为撒会觉得自己下蛋很费劲呢？我忽然想到写作本来就是很愿意做、写了心里舒服得到休息的事啊，当然我是指写流水账。写读书笔记和写小说这种费脑子的还是费劲，但写完了以后也还是感觉拥堵的心灵终于疏通了似的，属于过程难熬但是结果令人满足的，至于隔段时间再看觉得自己写得烂那又是后话了。

所以如果觉得下蛋很费劲，要么就是短暂失去了思考与写作的乐趣，提不起劲来写，人犯懒，要么就是觉得没撒好写的，兴趣转移了，总之就是二十年目睹写作之怪现象……

**Reply by yihui** [reply]
好呀么一个”二十年目睹写作之怪现象“，是要笑死我然后继承我的电脑看我未发布的博客吗。

是的，流水账是愿意写的，也毫不费劲；读书笔记和小说是费劲，但也有疏通效果。现在只是打工人撒完野刚回到厂里，暂时没多少时间。过去几周我感觉每周我都像干了以前一个月的活（当然，这只是表明我以前主要在摸鱼）。

我有一个久远的心愿，这几天狗不停爪地刨了几天，终于在昨天深夜把它大致完成了，就是这个网站主题：https://github.com/yihui/hugo-paged 接下来我要用这个主题翻新一下我的鸡窝，才能以更加舒服的姿势下蛋。

**Reply by earfanfan** [reply]
下蛋这个梗，在我记忆中最早是“高产如母猪下蛋”，所以你那是猪窝，哈哈哈。

这个新主题，感觉像是重新装修了，跟印象中园主的博客主题有一小丢丢像。

本阳间作息者是日常骑着小毛驴漫步踱过时间的间隙，而你这阴间作息者一旦拼命熬夜症间歇性发作，大概就是骑着白马快速跃过时间的间隙……啊，俺困得睁不开狗眼了，废话进入终结

**Reply by yihui** [reply]
好的，是翻新猪圈。

这新主题跟园主的网站主题除了一丢丢像之外（顶栏、正文、底栏），就是完全不像了。需要在电脑屏幕上看才能看出它四角上的书籍裁切标记，手机屏幕上我隐藏掉了这个标志性的元素。还有日志目录页上的虚线及“页码”，都是仿照书籍排版的感觉。

我把猪圈翻新完之后还有几篇英文博客要写，写完了再整理我浏览器里满满当当的标签页们，最后才有可能把耕博提上日程，所以这匹马还很有的跑。

**Reply by earfanfan** [reply]
新主题就是在电脑上看的，昨晚也看到那四个角了，但是并没有给我书籍排版的感觉，啊哈哈，给我第一感觉是很像 WORD 文档……

按你自己的节奏来吧。我是催耕不是催命哈，也不是催你写作业（ps今早上班路上还听见一个妈妈和一个儿子在争论小学生无作业日，那位妈妈说上了三年级就没有无作业日了）。

话说我昨晚突然觉得很多带有动物的成语很好玩，所以化用了白驹过隙。刚才突然想到可以查一下你的生肖，然后再化用一波，然而搜了搜没搜到有老鼠但是寓意较好的成语。唉，毕竟成语都是古人写的，来自于古人看待世界的态度，农耕社会的古人们看待许多生物都是从对人类有利有害的角度来看的，古人觉得老鼠这个意向不好，于是就没组多少寓意好的成语了吧，回头我再找找看。

话又说回来，我今早发现了一个字，是土话里的“抛”，如果说土地很“抛”，是指土地疏松，如果说头发很“抛”是指头发蓬松，如果说一个人变“抛”了，既可以指一个人身体上变胖了也可以指心理上飘了。感觉土话里的字，总是用起来更加活范，当然啦，汉字单个字含义确实广一些。在学语文课的过程中，似乎普通话里面几乎就是要把词的含义给圈定在一个小范围里固定起来，字一旦组词，含义就固定，明明字本身的含义其实更广一些。我想之所以变成这样，可能是因为人们要用词语串联成句子，而含义更固定的话，人们才可以根据固有且共同认可的标准来互相传达信息、解读信息，最终才更方便达到分工合作共同发展的效果吧。

**Reply by yihui** [reply]
猪圈已经翻新完毕（可能会受浏览器缓存影响，需要强制刷新以加载最新的 CSS 样式）。我已经太多年没打开过 Word 了，要是不提醒，我都已经忘了它里面有四个裁切标记了。本来我想的是我的那些 PDF 书里的标记。

“鼠”字大概是找不到什么好词的。查字典只看到“鼠思”可以勉强算中性词了（还是有些负面）。完全不记得每次鼠年过年的时候大家是怎么发祝福语的。

“抛”这个字[我几年前也提过](https://yihui.org/cn/2019/02/radish/)。我怀疑应该是“泡”。在土话里的意思确实比普通话里更活泛。

**Reply by earfanfan** [reply]
话说我平时有在豆瓣上收一些memes，这个词不知道咋念，中文名称应该叫做梗图。不过跟你不同的是，我看静态梗图多点。如下。
![微信图片_20230910210222](https://github.com/earfanfan/yuanfan/assets/29876335/a7ab6929-5c85-4356-81ba-f4c2e9fd2de5)
借这个梗图的意思。话说成语都是人造的，按理说我也是人呀，也可以造成语呀。虽然说古人造的成语更通用，但我造的不通用俺也不介意。

从前有一只豚鼠，性格很温和，所有动物都喜欢它，有一天这只豚鼠碰到了另一只性格温和的狗子，它俩成为了好朋友，一鼠一狗都想到了传说中白头如新、倾盖如故的典故，它们都觉得应该创造一个成语来纪念它们的相知相交，于是一个新的成语诞生了，生了，了……

**Reply by yihui** [reply]
meme 念“谜母”。

这独孤求认的大白鲨就是传说中的……鲨雕……么。

所以新的成语是啥呢？鼠狗之交？

**Reply by earfanfan** [reply]
完了，你把我带沟里了，本来我看到这条鲨鱼是觉得可爱的，这下再看只觉得鲨鱼是真鲨雕了……

新成语容我再想想吧，鼠狗之交刚也想到了，但是觉得太普通了，我要想个惊世骇俗、石破天惊、惊天动地、地动山摇、摇……什么的。

**Reply by liechi** [reply]
>meme 念“谜母”。

我一直念“么么”，后来我身边一些知道正确发音的也开始念么么了，于是成了meme现象在生活中的又一个鲜活例子。

**Reply by earfanfan** [reply]
@liechi

怎么梗图还能拿到日常生活中来交流的么，我觉得梗或者说迷因是个很神奇的东西，只能意会难以言传。

话说列弛你是属什么的？我是指生肖属相。兔子么？

**Reply by liechi** [reply]
它有比梗图更宽泛的意思。https://zh.wikipedia.org/wiki/%E8%BF%B7%E5%9B%A0

我属龙。
